onnxruntime/core/graph/function.cc

```diff
@@ -260,7 +273,7 @@ FunctionImpl::FunctionImpl(const onnxruntime::Graph& graph,
   const auto* node_in_parent_graph = parent_graph_->GetNode(node_index);
   op_schema_ = onnxruntime::make_unique<ONNX_NAMESPACE::OpSchema>();
   op_schema_->SetName(onnx_func_proto_.name());
-  op_schema_->SetDomain(onnx_func_proto_.node().Get(0).domain());
+  op_schema_->SetDomain(node_in_parent_graph->Domain());
```

Comment:
>node_in_parent_graph->Domain() [](start = 24, length = 30)

Just to clarify, this is so each node can bring its own domain?

Reply:
node in parent graph refers to the node for function Op. op_schema domain should be set to this node's domain (.i.e. function op domain). Current code assumes that the function op itself and the first node in the function body belong to same domain.